scripts/arm_data_converter.py

```diff
@@ -1008,16 +1008,16 @@
 
 # New epic weapons
 skillnamelist["epic-grandEpic"] = {
-    u"エピックブランド・ゲイン": "all",
+    u"エピックブランド・ゲイン": "dark",
```

Comment:
is this "dark" ? 2nd skill does not have element

Reply:
because all is reserved for weapons whose element gets changed when added to the arm list. The element selector gets into a wierd state when you throw "all" at him, it shows fire but fire is not selected. For non elemental stuff the default element is dark, so the selector is set to dark and not in an error state.